src/services/data_model_engine.js

```diff
@@ -213,23 +213,19 @@ export default class DataModelEngine {
     await this.entityRepository.discardBundling(bundleStubId);
   }
 
-  async uploadAcceptedBundleCandidates() {
+  async uploadAcceptedBundleCandidates(callbacks = {success: async () => {}, fail: async () => {}}) {
```

Comment:
I don't think it's a good idea to create new callbacks as a default parameter. Usually it's handled in a way that if callback is defined => call it

Reply:
Or maybe at least use Function() constructor?

Reply:
I like the way it is right now as it shows that the functions can be `async`. Even better would be to use TypeScript 😮